.github/workflows/livetest.yml.disabled

```diff
@@ -1,118 +0,0 @@
-          echo "::set-output name=allowed_user::true"
-
-    outputs:
-      allowed_user: ${{ steps.check_user_perm.outputs.allowed_user }}
-
-  live-test:
-    needs: pr-info
-    if: needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/livetest')
-    runs-on: ubuntu-20.04
-    env:
-      RELAY_HOST: logs.cockpit-project.org
-    steps:
-      - name: Install dependencies
-        run: |
-          sudo apt update
-          sudo apt install -y cockpit-system cockpit-ws nodejs npm make
-
-      - uses: actions/github-script@v3
-        id: get-pr
-        with:
-          script: |
-            const request = {
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              pull_number: context.issue.number
-            }
-            core.info(`Getting PR #${request.pull_number} from ${request.owner}/${request.repo}`)
-            try {
-              const result = await github.pulls.get(request)
-              return result.data
-            } catch (err) {
-              core.setFailed(`Request failed with error ${err}`)
-            }
-
-      - uses: actions/checkout@v2
-        with:
-          repository: ${{ fromJSON(steps.get-pr.outputs.result).head.repo.full_name }}
-          fetch-depth: 0
-          ref: ${{ fromJSON(steps.get-pr.outputs.result).head.ref }}
-
-      - name: Compile and install the project
-        run: make && sudo make install
-
-      - name: Create a new test user
-        # WARNING: if you want to add this users to sudoers make sure to properly delete the SINK_SSHRELAY_PRIVATE
-        run: sudo useradd --create-home livetest; echo livetest:foobar | sudo chpasswd
-
-      - name: Get the commit SHA that triggered the workflow
-        id: get_commit_sha
-        run: |
-          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
-
-      - name: Get ssh gateway port to forward to
-        id: remote_port
-        env:
-          SHA_SHORT: "${{ steps.get_commit_sha.outputs.sha_short }}"
-        run: |
-            echo ::set-output name=SSH_GATEWAY_PORT::$((16#$SHA_SHORT % 32768 + 1500))
-
-      - name: Write a comment on the PR with link to the cockpit instance
-        uses: actions/github-script@v3
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            github.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: 'Cockpit preview: https://${{ env.RELAY_HOST }}:${{ steps.remote_port.outputs.SSH_GATEWAY_PORT }}'
-            })
-
-      - name: Create an SSH private key from the secret
-        run: |
-          sudo touch /run/SINK_SSHRELAY_PRIVATE && sudo chown $(id -u):$(id -g) /run/SINK_SSHRELAY_PRIVATE
-          echo "${{ secrets.SINK_SSHRELAY_PRIVATE }}" > /run/SINK_SSHRELAY_PRIVATE
```

Comment:
If we ever bring this back, let's use a deploy key instead of an org-wide secrets. The former are much easier to handle and refresh.